diff --git a/node_modules/@capacitor/camera/ios/Sources/CameraPlugin/CameraPlugin.swift b/node_modules/@capacitor/camera/ios/Sources/CameraPlugin/CameraPlugin.swift
index 968733e..2dbab70 100644
--- a/node_modules/@capacitor/camera/ios/Sources/CameraPlugin/CameraPlugin.swift
+++ b/node_modules/@capacitor/camera/ios/Sources/CameraPlugin/CameraPlugin.swift
@@ -219,8 +219,8 @@ public class CameraPlugin: CAPPlugin, CAPBridgedPlugin {
                         }
                     }
 
-                    group.notify(queue: .global(qos: .utility)) { [weak self] in
-                        self?.returnImages(processedImages)
+                    group.notify(queue: .global(qos: .utility)) {
+                        self.returnImages(processedImages)
                     }
                 }
             } else {
